powersimdata/input/check.py

```diff
@@ -32,6 +35,15 @@ def check_grid(grid):
             error_messages.append(
                 f"Exception during {check.__name__}: {sys.exc_info()[1]!r}"
             )
+    # Run checks which operate on a pandas data frame
+    for gencost_key in ("before", "after"):
+        try:
+            _check_gencost(grid.gencost[gencost_key], error_messages)
+        except Exception:
+            error_messages.append(
+                f"Exception during {check.__name__} {gencost_key}: "
+                "{sys.exc_info()[1]!r}"
```

Comment:
Not sure if this needs an "f" on the second line

Reply:
You're right.

Reply:
Done.